match/lib/match/generator.rb

```diff
@@ -39,7 +39,13 @@ def self.generate_provisioning_profile(params: nil, prov_type: nil, certificate_
 
       prov_type = Match.profile_type_sym(params[:type])
 
-      profile_name = ["match", profile_type_name(prov_type), app_identifier].join(" ")
+      names = ["match", profile_type_name(prov_type), app_identifier]
+
+      if params[:platform] != 'ios' # For ios we do not include the platform for bc
```

Comment:
mmm. I understood what you need, what I didn't realize was that "bc" was the abbreviation for Backward Compatibility. It might be ovious in the context of the patch, but standalone, might not be understandable at first sigh.

Please replace the abbreviation with the full words.